### ISSUE TITLE
Add missing mouse hover focus indicators to show app details page

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#213]
@krozett The following are missing mouse hover focus indicators on show app details page
a.	Categories link
![image](https://user-images.githubusercontent.com/1034174/108131281-e7544700-7065-11eb-90a0-7c81a203f1b5.png)
b.	Rating score and submit review button
![image](https://user-images.githubusercontent.com/1034174/108131331-f5a26300-7065-11eb-8872-2f80d6e31add.png)

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
